com.unity.render-pipelines.high-definition/Documentation~/snippets/shader-graph-blocks/cuticle-angle.md

```diff
@@ -0,0 +1,6 @@
+<tr>
+<td>**Smoothness**</td>
+<td>The angle in degrees at which the scales on a hair fiber are tilted from the strand direction. (for human hair, typically between 2 to 3 degrees). Use this property to “shift” the highlight..</td>
```

Comment:
<td>The angle (in degrees) that the scales on a hair fiber tilt from the strand direction. For human hair, this value is usually between 2 to 3 degrees. Use this property to “shift” the highlight.</td>